pkg/parse/state.go

```diff
@@ -99,70 +109,70 @@ const retryLimit = 12
 //	4m24.712222056s, 9m18.97652295s, 17m15.344384599s, 35m15.603237976s.
 func defaultBackoff() wait.Backoff {
```

Comment:
Might be cleaner to move this default code into ResetRetryBackoffAndTimer and then always call ResetRetryBackoffAndTimer when you want to reset the retry backoff.

Then while you're at it, also move the NewTimer initialization into ResetRetryBackoffAndTimer so it doesn't panic when nil.